src/v2/styles/MessageInput/MessageInput-layout.scss

```diff
@@ -72,6 +72,7 @@
         min-height: 0;
         max-height: 100%;
         display: flex;
+        align-items: center;
```

Comment:
Looking at the [design doc](https://www.figma.com/file/oeHiSMDoutuVbJ9WiVyhcG/Chat-UI-Kit-1.5-Android-%26-Web?type=design&node-id=21547-390045&mode=design&t=8wvb0wutRX71snTt-0), looks like the emoji picker button should actually stick to the bottom of the input. WDYT, should we align our styles with that design?

Reply:
That's fair, I haven't checked that. The problem is that in Angular the emoji picker can be added by a [code example](https://getstream.io/chat/docs/sdk/angular/code-examples/emoji-picker/) and I wanted to avoid breaking existing CSS (I'm working on a [bugfix](https://github.com/GetStream/stream-chat-angular/issues/579) currently). 

After the bugfix this is how existing integrations look like on some browsers - the icon is slightly higher than the placeholder, this was the problem I wanted to fix.
![Screenshot 2024-04-03 at 11 15 10](https://github.com/GetStream/stream-chat-css/assets/6690098/8b9fc71e-fcec-442a-bfd6-2977cd3d7d1e)

I think we can only solve this from the SDK reliably by centering stuff, but I'll make it Angular-only. Integrators still be able to position the emoji picker icon wherever they like

Reply:
hm, never mind, the same bug exists currently, and `align-items: center` doesn't fix it :D I'll close the PR